src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,72 @@
+package com.elixr.training;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+                if(args.length!=2)
+                {
+                    System.out.println("enter the valid path");
+                    return;
+                }
+                String inputfile = args[0];
+                String wordsearch = args[1];
+                File file = new File(inputfile);
+                supportfile(file);
+                contains(file);
+                System.out.println("processing................");
+                String data = null;
+                data = readFile(inputfile);
+                wordsearch(data,wordsearch);
```

Comment:
wordSearch method will be called even if file is not a txt or if file is empty. This is not needed. Only call wordSearch if the file is valid and has contents

Reply:
stopped the loop when file is not in supported format using exit method